### ISSUE TITLE
Radio two-way binding with `undefined` option doesn't work as expected

### ISSUE THREAD

**Issue by marcalexiei** [ractivejs/ractive#3309]
## Description:
Me and @giovannipiller ran into another interesting issue 😄.
Two-way with radios and an `undefined` value doesn't update target expression right away if previous value was a string.

## Versions affected:
1.3.6

## Platforms affected:
Tested on Chrome and Safari

## Reproduction:
https://jsfiddle.net/fLu74c2j/4/

1. Load the page (`currentValue` is `something `)
2. Click undefined option

**Actual behaviour:**
`currentValue` remains `something`

**Expected behaviour:**
`currentValue` change to `undefined`

**Additional note:**
Clicking `another string` before clicking `undefined` will work as expected.

```javascript
const r = window.r = new Ractive({
  el: '#main',
  template: `
  {{#each list}}
    <label>
    	<input type="radio" name="{{~/currentValue}}" value="{{.value}}">
    	{{.label}}
    </label>
    <br>
  {{/each}}
  
  <br>
  
  currentValue:
  {{#if typeof currentValue === 'string'}}
  	<pre>'{{currentValue}}'</pre> 
  {{else}}
  	<pre>undefined</pre> 
  {{/if}}
  `,
  data: {
    currentValue: 'something',
    list: [{
        value: undefined,
        label: 'undefined (selecting this right away does not change currentValue)'
      },
      {
        value: 'something',
        label: 'string'
      },
      {
        value: 'something2',
        label: 'another string'
      },
    ],
  },
});
```

**Comment by giovannipiller** [comment]
Setting `currentValue` directly via JS works as expected, so issue is likely related to two-way bindings.
```javascript
oncomplete() {
  this.set({ currentValue: undefined });
}
```

**Comment by evs-chris** [comment]
Yep, it was a problem with initialization of the tracking value on radio bindings in a group. `undefined` is the _one_ case that would fail :smile:. This should now be fixed on edge and will get backported for 1.2 and 1.3 in a bit. Thanks!
